test/yki/handler/auth_test.clj

```diff
@@ -102,12 +122,43 @@
     (let [handler (create-routes port)
           session (peridot/session handler)
           response (-> session
+                       (peridot/request (str routing/auth-root "?success-redirect=/yki/auth/user"))
                        (peridot/request (str routing/auth-root routing/auth-init-session-uri)
-                                        :headers (json/read-value (slurp "test/resources/headers2.json" :encoding "ISO-8859-1"))
-                                        :request-method :get)
+                                        :headers (json/read-value (slurp "test/resources/headers2.json" :encoding "ISO-8859-1")))
                        (peridot/follow-redirect))
```

Comment:
Usually when I see anything besides UTF I start feeling weird, but we can talk about that at the office :)